snowflake_data_profiler/profiling/profiler.py

```diff
@@ -32,7 +32,9 @@ def file_to_df(file_name):
         df = pd.read_sql(file_name)
     return df
 
-def connect_to_snowflake(sfUser, sfPswd, sfURL, sfDatabase, sfSchema, sfTable, sfWarehouse=None, sfRole=None):
+
+# establishes snowflake connection
+def establish_connection(sfUser, sfPswd, sfURL, sfDatabase, sfSchema, sfTable, sfRole=None):
```

Comment:
This is a much better name. As you read more code, you'll see repeatable patterns that people use for this kind of thing.

The fact that this function returns the connection object means I'd name it `get_snowflake_connection`. This would be a fine function if it performed a side effect only, but this is a `getter` method.